packages/preview-middleware-client/types/sap.ui.d.ts

```diff
@@ -1,4 +1,5 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 declare namespace sap.ui {
     export const version: string;
+    export function requireSync(module: string): any;
```

Comment:
Where is this used? If its not then it should be removed.

Reply:
fixed in #[1c037bc](https://github.com/SAP/open-ux-tools/pull/1941/commits/1c037bc26af2f5b88ee97d4a7563060b8fe654a6)